### PR TITLE
Include GYP test

### PR DESCRIPTION
**Pull request by uilianries**
Signed-off-by: Uilian Ries <uilianries@gmail.com>

### CODE REVIEW

File: test_package/conanfile.py [hunk 1]
```diff
@@ -4,9 +4,26 @@
 from conans import ConanFile
 
 
+TEST_PACKAGE_GYP = """{
+    'targets': [
+      {
+        'target_name': 'test_package',
+        'type': 'executable',
+        'sources': [
+          'test_package.cpp',
```
Comment (SSE4) [inline test_package/conanfile.py:13]: forget to commit this file?

**Review by SSE4 — CHANGES_REQUESTED** [review]

File: test_package/conanfile.py (hunk 1 above)
Comment (uilianries) [reply test_package/conanfile.py:13]: It wasn't copied to build folder :thinking: So I created the file using python. Any idea?

Comment (SSE4) [reply test_package/conanfile.py:13]: you have created `test_package.gyp`, right?
but what about `test_package.cpp`?

Comment (uilianries) [reply test_package/conanfile.py:13]: oh, it doen't care, because i'm just testing make generator

Comment (uilianries) [reply test_package/conanfile.py:13]: done!

**Review by SSE4 — APPROVED** [review]

**Pull request merged**